lib/src/Uncertainty/Distribution/CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ ot_add_source_file (UniformFactory.cxx)
 ot_add_source_file (UserDefined.cxx)
 ot_add_source_file (UserDefinedFactory.cxx)
 ot_add_source_file (VonMises.cxx)
+ot_add_source_file (VonMisesFactory.cxx)
```

Comment:
please mention issue in ChangleLog + new class also in ChangeLog

Reply:
also mention the issue in the commit message